contracts/src/v0.8/dev/Operator.sol

```diff
@@ -306,7 +304,7 @@ contract Operator is
     public
     validateAuthorizedSenderSetter()
   {
-    TargetsUpdatedAuthorizedSenders(targets, senders, msg.sender);
+    emit TargetsUpdatedAuthorizedSenders(targets, senders, msg.sender);
```

Comment:
woah how did this work before?

Reply:
I think previous Solidity versions allowed it but threw a warning.